app/views/docs/index.phtml

```diff
@@ -87,7 +87,7 @@ $cols = [
                     <li><a href="/docs/pagination">Pagination</a></li>
                     <li><a href="/docs/custom-domains">Custom Domains</a></li>
                     <li><a href="/docs/permissions">Permissions</a></li>
-                    <li><a href="/docs/error-codes">Error Codes</a></li>
+                    <li><a href="/docs/response-codes">Response Codes</a></li>
```

Comment:
reminder to update this file as well
https://github.com/appwrite/homepage/blob/cc7e8ddcbc3bf401e16e34cf8f46a2d97b65400c/app/controllers/home.php#L538-L541